plugins/lsp/language-servers/default.nix

```diff
@@ -275,19 +274,18 @@ let
       name = "ltex";
       description = "ltex-ls for LanguageTool";
       package = pkgs.ltex-ls;
-      settingsOptions = import ./ltex-settings.nix { inherit lib helpers; };
-      settings = cfg: { ltex = cfg; };
+      settingsOptions = import ./ltex-settings.nix {inherit lib helpers;};
+      settings = cfg: {ltex = cfg;};
     }
     {
       name = "lua-ls";
       description = "lua-ls for Lua";
       # Use the old name of the lua LS if the user is on a stable branch of nixpkgs
       # Rename occurred here: https://github.com/NixOS/nixpkgs/pull/215057
       package =
-        if (hasAttr "lua-language-server" pkgs) then
-          pkgs.lua-language-server
-        else
-          pkgs.sumneko-lua-language-server;
+        if (hasAttr "lua-language-server" pkgs)
+        then pkgs.lua-language-server
+        else pkgs.sumneko-lua-language-server;
```

Comment:
Strange that unrelated code needs to be formatted...

Was this definitely `nix fmt` and not some additional formatter you have in your editor/workflow? The result looks like [Alejandra](https://github.com/kamadorueda/alejandra).

If this is coming from `nix fmt`, maybe rebase into a separate commits; one doing `nix fmt` and another adding this PR's changes.

Reply:
> Looks like the hook is fighting the nix fmt..

Interesting

Reply:
I had to direnv reload so it would reinstall the pre-commit-hook. It must have changed.

Reply:
I can't see why there'd be an issue; The only difference is `nix fmt` is wrapped with `treefmt` while `pre-commit` is directly using `statix` and `nixfmt-rfc-style`, which `treefmt` is _also_ configured to use... ([here](https://github.com/nix-community/nixvim/blob/9d858de2e9ab136d1c53d92af62fed8fccf492ab/flake-modules/dev/default.nix#L12-L36))

Reply:
> Was the repo using Alejandra before and switched to new nixfmt-rfc-style?

Yeah, it was

> I had to direnv reload so it would reinstall the pre-commit-hook.

Makes sense, didn't consider you might have had an old hook installed :smile: